rewrite-groovy/src/main/java/org/openrewrite/groovy/GroovyParserVisitor.java

```diff
@@ -1108,6 +1108,7 @@ public void visitBlockStatement(BlockStatement block) {
         public void visitCatchStatement(CatchStatement node) {
             Space prefix = sourceBefore("catch");
             Space parenPrefix = sourceBefore("(");
+            List<J.Modifier> modifiers = getModifiers();
```

Comment:
We don't have to guard here against any non `final` (`static` a.o.) modifier because the Groovy toolchain would reject the code as invalid beforehand, right?

Reply:
Yes!